app/test/frontend/golden/authorized_page.html

```diff
@@ -15,6 +15,7 @@
     <meta itemprop="image" content="/static/img/dart-logo-400x400.png?hash=mocked_hash_58383222"/>
     <meta property="og:image" content="/static/img/dart-logo-400x400.png?hash=mocked_hash_58383222"/>
     <link rel="shortcut icon" href="/favicon.ico?hash=mocked_hash_985685822"/>
+    <link rel="search" type="application/opensearchdescription+xml" title="Dart packages" href="/static/osd.xml?hash=mocked_hash_869379763">
```

Comment:
You'll need to close the element.
Btw. what's the error message you get on Windows running this test?

Reply:
The `setUpAll` fails because it tries to run `pub get` and can't find the executable on Windows. It is anecdotal but I think I have a fairly typical Windows setup; a workaround I figured out in other projects was to use the `runInShell` flag when running the process. Maybe there's something odd about how Dart looks at the Windows path such that the shell does a better job? I dunno.

Even if that were fixed, though, the next line runs a `build.sh` and that will fail regardless. I tried skipping the setup step of the tests altogether but most things then fail because nothing gets built.